rllib/agents/trainer_config.py

```diff
@@ -866,10 +869,11 @@ def offline_data(
             input_evaluation: Specify how to evaluate the current policy. This only has
                 an effect when reading offline experiences ("input" is not "sampler").
                 Available options:
-                - "wis": the weighted step-wise importance sampling estimator.
-                - "is": the step-wise importance sampling estimator.
-                - "simulation": run the environment in the background, but use
+                - "simulation": Run the environment in the background, but use
                 this data for evaluation only and not for learning.
+                - Any subclass of OffPolicyEstimator, e.g.
+                ray.rllib.offline.is_estimator::ImportanceSamplingEstimator or your own
```

Comment:
Can we document the available options, though? what are is and wis and when would I use each?